src/run.js

```diff
@@ -193,6 +193,7 @@ const processPage = ({
       options.withoutjavascript === undefined
         ? true
         : !!options.withoutjavascript;
+	const isJavaScriptEnabled = (options.isJavaScriptEnabled === undefined) || !!options.isJavaScriptEnabled;
```

Comment:
I see that you used the same pattern as for `const withoutjavascript = ...`  but I can't at all remember why it was done like that. A bunch more idiomatic style would be:
```javascript
const isJavaScriptEnabled = !!options.isJavaScriptEnabled;
```

As a matter of fact, a bunch of these lines above can be simplified. E.g.
```javascript
const debug = !!options.debug;
```
...is way better because it guarantees that the variable is definitely a boolean. 

I.e.

```javascript
>>> var options = {debug: []}
>>> const debug = options.debug || false
>>> debug
[]
```

Reply:
Now when isJavaScriptEnabled option is undefined script must be enabled on page.
I can rename this to disableJavaScript and after that const disableJavaScript = !!options.disableJavaScript;